packages/@react-spectrum/filetrigger/docs/FileTrigger.mdx

```diff
@@ -0,0 +1,162 @@
+{/* Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/filetrigger';
+import typesDocs from 'docs:@react-types/shared/src/events.d.ts';
+import {PropTable, HeaderInfo, TypeLink, PageDescription, StateTable} from '@react-spectrum/docs';
+import styles from '@react-spectrum/docs/src/docs.css';
+import packageData from '@react-spectrum/filetrigger/package.json';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import {Divider} from '@react-spectrum/divider';
+import {ExampleCard} from '@react-spectrum/docs/src/ExampleCard';
+import {Keyboard} from '@react-spectrum/text';
+import Button from '@react-spectrum/docs/pages/assets/component-illustrations/ActionButton.svg';
+import Link from '@react-spectrum/docs/pages/assets/component-illustrations/Link.svg';
+
+---
+category: Buttons
+---
+
+# FileTrigger
+
+<PageDescription>{docs.exports.FileTrigger.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['FileTrigger']} />
+
+## Example
+
+```tsx example
+import {FileTrigger, Button} from '@adobe/react-spectrum';
```

Comment:
Isn't our pattern to use `@react-spectrum/dropzone.mdx` and have the docs build replace it with `@adobe/react-spectrum`?

Reply:
oh im not sure. do you mind clarifying what you mean? i checked some other doc examples and they seem to import from `'@adobe/react-spectrum'` directly

Reply:
I don't think it really matters either way, just so long as the final result is that the import shows the mono package